components/common/listingCart/ListingCartModal.vue

```diff
@@ -0,0 +1,196 @@
+          <ListingCartPriceInput
+            v-model="fixedPrice"
+            check
+            class="pt-2"
+            @confirm="setFixedPrice" />
+        </div>
+        <div class="py-2">
+          <ListingCartItem
+            v-for="nft in listingCartStore.itemsInChain"
+            :key="nft.id"
+            :nft="nft" />
+        </div>
+        <div
+          class="is-flex border-top is-justify-content-space-between py-4 px-6">
+          Potential Earnings
+          <div class="is-flex">
+            <CommonTokenMoney :value="totalNFTsPrice" class="has-text-grey" />
+            <span class="has-text-weight-bold ml-2"> ${{ priceUSD }} </span>
+          </div>
+        </div>
+
+        <div class="is-flex is-justify-content-space-between pb-5 px-6">
+          <NeoButton
+            :disabled="!!listingCartStore.incompleteListPrices"
+            :label="confirmListingLabel"
+            variant="k-accent"
+            no-shadow
+            class="is-flex is-flex-grow-1 btn-height"
+            @click.native="confirm" />
+        </div>
+      </div>
+    </NeoModal>
+  </div>
+</template>
+<script setup lang="ts">
+import { Interaction } from '@kodadot1/minimark/v1'
+import ListingCartPriceInput from '@/components/common/listingCart/ListingCartPriceInput.vue'
+import { totalPriceUsd } from '@/components/common/shoppingCart/utils'
+import IdentityItem from '@/components/identity/IdentityItem.vue'
+import CommonTokenMoney from '@/components/shared/CommonTokenMoney.vue'
+import { NeoButton, NeoModal } from '@/libs/ui'
+import { usePreferencesStore } from '@/stores/preferences'
+import { TokenToList } from '@/composables/transaction/types'
+import { useListingCartStore } from '@/stores/listingCart'
+import { calculateBalance } from '@/utils/format/balance'
+import { sum } from '@/utils/math'
+import { warningMessage } from '@/utils/notification'
+const { isLogIn, accountId } = useAuth()
+const { urlPrefix } = usePrefix()
+const preferencesStore = usePreferencesStore()
+const listingCartStore = useListingCartStore()
+const { transaction, isLoading, status } = useTransaction()
+const { $i18n } = useNuxtApp()
+
+const fixedPrice = ref()
+function setFixedPrice() {
+  listingCartStore.setFixedPrice(fixedPrice.value)
+  fixedPrice.value = undefined
+}
+
+const priceUSD = computed(() => {
+  const { nfts, royalties } = totalPriceUsd(listingCartStore.itemsInChain)
+  return (nfts + royalties).toFixed(2)
+})
+
+const totalNFTsPrice = computed(() =>
+  sum(listingCartStore.itemsInChain.map((nft) => Number(nft.price)))
+)
+
+const confirmListingLabel = computed(() => {
+  switch (listingCartStore.incompleteListPrices) {
+    case 0:
+      return 'Complete Listing'
+    case 1:
+      return '1 Item Is Missing Price'
+    default:
+      return `${listingCartStore.incompleteListPrices} Items Are Missing Price`
+  }
+})
+async function confirm() {
+  const token = [] as TokenToList[]
+  for (const item of listingCartStore.itemsInChain) {
```

Comment:
please use ES6 method `forEach`

Reply:
I just followed STYLE_GUIDE
<img width="1078" alt="image" src="https://github.com/kodadot/nft-gallery/assets/3810177/98b64ab8-8b4b-4edd-b929-5ba6484c1ba4">

But yes, I can change it to forEach here